[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongos-k8s/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongos-k8s/0 [idle] active: 
  mongos-k8s/1 [executing] maintenance: starting mongos
[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongos-k8s/0 [idle] active: 
  mongos-k8s/1 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:834 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.5.3    unsupported  01:36:37Z

App            Version  Status  Scale  Charm        Channel  Rev  Address         Exposed  Message
config-server           active      1  mongodb-k8s  6/edge    59  10.152.183.111  no       
mongos-k8s              active      2  mongos-k8s              0  10.152.183.86   no       
shard0                  active      1  mongodb-k8s  6/edge    59  10.152.183.118  no       

Unit              Workload  Agent  Address     Ports  Message
config-server/0*  active    idle   10.1.1.142         Primary
mongos-k8s/0*     active    idle   10.1.1.136         
mongos-k8s/1      active    idle   10.1.1.143         
shard0/0*         active    idle   10.1.1.141         Primary

[32mINFO    [0m pytest_operator.plugin:plugin.py:840 Juju error logs:

unit-config-server-0: 01:31:42 ERROR unit.config-server/0.juju-log config-server:6: Failed to add shard shard0 to the config server, error=OperationFailure("Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1729387902, 2), 'signature': {'hash': b'\\xe4r\\x1e\\x12\\n\\xedSMm\\xa2\\x85\\xf1B\\xcb \\xa6\\xdc}\\xac\\xcd', 'keyId': 7427664240669884441}}, 'operationTime': Timestamp(1729387902, 2)}")
unit-config-server-0: 01:31:42 ERROR unit.config-server/0.juju-log config-server:6: shard0 shard does not have the same auth as the config server.
unit-mongos-k8s-0: 01:33:24 ERROR unit.mongos-k8s/0.juju-log cluster:7: mongos service is not ready yet.
unit-mongos-k8s-0: 01:34:26 ERROR unit.mongos-k8s/0.juju-log cluster:7: mongos service is not ready yet.

[32mINFO    [0m pytest_operator.plugin:plugin.py:904 Forgetting main...